src/main/java/com/microsoft/azure/functions/worker/broker/JavaFunctionBroker.java

```diff
@@ -93,7 +94,7 @@ private void initializeFunctionInstanceInjector() {
 			Iterator<FunctionInstanceInjector> iterator = ServiceLoader.load(FunctionInstanceInjector.class).iterator();
 			if (iterator.hasNext()) {
 				this.functionInstanceInjector = iterator.next();
-				WorkerLogManager.getSystemLogger().info("Load function instance injector: " + this.functionInstanceInjector.getClass().getName());
+				WorkerLogManager.getSystemLogger().log(Level.INFO, "Load function instance injector: {0}" + this.functionInstanceInjector.getClass().getName());
```

Comment:
some how this logs something like "Load function instance injector: {0}" in kusto, not log out the class name. I am thinking to switch to using String.format, once we figure out why it not works and fix it, we can have it change back into this way.